src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -387,7 +389,23 @@ public MappingsSaver mappingsSaver() {
 
     @Override
     public Notifier notifier() {
-        return new ConsoleNotifier(verboseLoggingEnabled());
+        String notifier = (String) optionSet.valueOf(NOTIFIER);
+        if ("ConsoleNotifier".equals(notifier)) {
+            return new ConsoleNotifier(verboseLoggingEnabled());
+        }
+        if ("Slf4jNotifier".equals(notifier)) {
+            return new Slf4jNotifier(verboseLoggingEnabled());
+        }
+        else {
+            try {
+                ClassLoader loader = Thread.currentThread().getContextClassLoader();
```

Comment:
it will be better to use well-known and tested approach. See ClassUtils from Apache Commons. https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ClassUtils.html . The dependency is already in the project.
```
compile 'org.apache.commons:commons-lang3:3.7'
```

Reply:
This part I took from httpServerFactory. I will refactor mine so it can serve as an example for the factory.